metecho/api/sf_run_flow.py

```diff
@@ -325,6 +325,7 @@ def run_flow(*, cci, org_config, flow_name, project_path, user):
         traceback = p.stdout.decode("utf-8")
         logger.warning(traceback)
         raise Exception(_last_line(traceback) or _last_line(err))
+    return out
```

Comment:
@wlonk Notice that this code is detecting a non-zero returncode and raising an exception on line 327 (the exception message ends up getting displayed on the frontend). I want to be sure the log is stored in that case too -- in fact that's the case when we are likely to go looking for the log.

And then we need to talk about the source for the log. It should be read from the log file I mentioned in the trello card, not from the cci process's stdout. The log file includes a copy of both stdout and stderr, as well as some additional context such as what command was run.